### PR TITLE
Fix: dimensionality misspelling

### PR DESCRIPTION
**Pull request by anslpa**
Dimensionality misspelling fixed.

### CODE REVIEW

**Comment by anslpa** [comment]
Ready to be merged. The Python DPF-Post (ansys-dpf-post) module won't be impacted.

File: ansys/dpf/core/dimensionality.py [hunk 1]
```diff
@@ -1,13 +1,13 @@
 """
-Dimensionnality
+Dimensionality
 ===============
```
Comment (akaszynski) [inline ansys/dpf/core/dimensionality.py:3]:
```suggestion
==============
```

File: ansys/dpf/core/dimensionality.py [hunk 2]
```diff
@@ -41,17 +41,17 @@ def __str__(self):
     
     @staticmethod
     def scalar_dim():
-        """Dimensionnality instance corresponding to a scalar field
+        """Dimensionality instance corresponding to a scalar field
 
         Returns
         -------
-        dimensionnality : Dimensionnality
+        dimensionality : Dimensionality
```
Comment (akaszynski) [inline ansys/dpf/core/dimensionality.py:48]: We should consider spelling out the class return types like:

```suggestion
        :class:`ansys.dpf.core.dimensionality.Dimensionality`
```

One of the things we've been doing within `pyaedt` is allowing sphinx to provide links between of return types and class documentation.  It's super helpful.  Also, there's no reason to spell out the parameter name of the return type.

Comment (anslpa) [reply ansys/dpf/core/dimensionality.py:48]: Hi @akaszynski, dpf core documentation already provides links between the return type and the considered class (working with this spelling syntax)

**Review by rlagha — APPROVED** [review]

**Pull request merged**